components/src/a11y/semantic-enrich/semantic-enrich.js

```diff
@@ -5,6 +5,8 @@ import Sre from '../../../../js/a11y/sre.js';
 import {EnrichHandler} from '../../../../js/a11y/semantic-enrich.js';
 import {MathML} from '../../../../js/input/mathml.js';
 
+export {Sre};
```

Comment:
I don't remember.  It may have been because it seemed that SRE was a natural thing to make available from the semantic-enrichment module.  I will remove it.